src/pages/components/card/card.jsx

```diff
@@ -0,0 +1,38 @@
+
+
+//import cardImgage from "../../assets/images/MensFashion.jpeg";
+
+
+export default function Card ({name,categoryName,price,discountedPrice,discount,stockDetail,cardImage}){
+return(
+    <div className="container1">
+              <div>
+                <img className="ecom-img1" src={cardImage} alt="ecomimage" />
+                <span className="card-badge1"> {stockDetail} </span>
```

Comment:
the top left border radius is being messed up by this badge maybe add the border radius to badge too?